skins/cat17/package.json

```diff
@@ -9,10 +9,13 @@
   },
   "scripts": {
     "watch": "gulp watch",
+    "watch-faq": "watchify -vd -p browserify-hmr -e faq-page/src/main.js -o ../../web/skins/cat17/scripts/faq.js",
+    "build-faq": "cross-env NODE_ENV=production browserify -g envify -p [ vueify/plugins/extract-css -o ../../web/skins/cat17/css/faq.css ] -e faq-page/src/main.js | uglifyjs -c warnings=false -m > ../../web/skins/cat17/scripts/faq.js",
+    "lint": "eslint --ext .js,.vue src",
```

Comment:
This will produce thousands of errors. Maybe it's because of the wrong coding style, but I guess we also need to restrict this to the `faq-page` directory for now.